src/TourManager/Repository/Entities/Client.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Travely.TourManager.Repository.Repositories.Entities
+{
+    public class Client
+    {
+        public long Id { get; set; }
```

Comment:
Create BaseEntity and move Id to to that Class , also add CreatedAt and UpdatedAt fields in BaseEntity